.github/workflows/codeql-analysis.yml

```diff
@@ -23,18 +23,14 @@ jobs:
       security-events: write
 
     steps:
-    -
-      name: Checkout repository
-      uses: actions/checkout@v3
-    # Initializes the CodeQL tools for scanning.
-    -
-      name: Initialize CodeQL
-      uses: github/codeql-action/init@v2
-      with:
-        languages: 'python'
-    -
-      name: Autobuild
-      uses: github/codeql-action/autobuild@v2
-    -
-      name: Perform CodeQL Analysis
-      uses: github/codeql-action/analyze@v2
+      - name: Checkout repository
+        uses: actions/checkout@v3
+      # Initializes the CodeQL tools for scanning.
+      - name: Initialize CodeQL
+        uses: github/codeql-action/init@v2
+        with:
+          languages: "python"
+      - name: Autobuild
+        uses: github/codeql-action/autobuild@v2
+      - name: Perform CodeQL Analysis
+        uses: github/codeql-action/analyze@v2
```

Comment:
I kind of like the old format, with the space to break up the steps. We can set some of the rules with:

https://github.com/adrienverge/yamllint

Add the config to stickler: https://stickler-ci.com/docs#yaml